elyra/pipeline/parser.py

```diff
@@ -198,3 +206,14 @@ def _get_parent_operation_links(node: Dict, embedded_node_id: Optional[str] = No
                 else:
                     links.extend(PipelineParser._get_input_node_ids(node_input))
         return links
+
+    @staticmethod
+    def _scrub_list(dirty: Optional[List]) -> List:
+        """
+        Clean an existing list by filtering out None and empty string values
+        :param dirty: a List of values
+        :return: a clean list without None or empty string values
+        """
+        if not dirty:
+            return []
+        return [clean for clean in dirty if clean]
```

Comment:
The comments reminded me that this will remove `None` and _falsey_ values (e.g., empty strings).  Do we want to remove empty strings?  Since this method was meant for generic lists, it will also remove '0' and 'False' values from integer and boolean lists, which we definitely do not want to do.

Since empty string values weren't problematic before this change, this is a departure from what worked yesterday (in cases where empty strings may have been included). 
```suggestion
        return [clean for clean in dirty if clean is not None]
```
Or we should change the signature (and method name) to indicate it only deals with string lists.  If so, the typing would be: `dirty: Optional[List[Optional[str]]]) -> List[str]`

Reply:
I think the string signature seems better...